### PR TITLE
Add and/or update documentation alt text

### PR DESCRIPTION
**Pull request by Carreau**
@isabela-pf originally said:

# Description

This is the second PR of its kind! If it helps, the previous changes from the previous event can be found at #4375

This PR adds or updates existing documentation alt text to be more descriptive. In particular, it describes the images in relation to the surrounding text they illustrate rather than describing them more generally. You may also find help in our discussion on the [collaborative working PR](https://github.com/isabela-pf/napari/pull/5).

We did not get to add or update all alt text in the documentation; this PR contributes only a selection.

## Type of change

This is a change to images in the documentation, but does not require a documentation update as far as I know.

# References

Continues fixing #3540

This work was done as a part of an event! You can find more info about the event and what resources we used on the [alt text workshop agenda](https://hackmd.io/rz-GeCi9RXixQdn5d4DEPg?both).

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change

## Final checklist:
- [x] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/docs/dev/guides/translations.html).

### CODE REVIEW

**Comment by Carreau** [comment]
This is a cross-repository port of https://github.com/napari/napari/pull/5240 – All credit due there – I'm even happy to close this PR and have @isabela-pf re-open it to give her the credit. I"ll describe how I did the port in the other PR.

**Comment by Carreau** [comment]
@isabela-pf if you want to retake over this PR, let me know I can guide you through the steps.

**Comment by isabela-pf** [comment]
Thank you so much, @Carreau! I'm fine with this PR being merged as-is, so let me know if there's  anything else I need to do here.

(Apologies for missing the docs move, that's totally my mistake!)

File: docs/howtos/layers/shapes.md [hunk 1]
```diff
@@ -357,8 +357,7 @@ You can delete vertices to a selected shape using the vertex deletion tool which
 can be selected either clicking on the vertex deletion tool in the layer
 controls panel or pressing the `X` key while the shapes layer is selected.
 
-For example see below: ![image: shape vertex
-editing](../../images/shape_vertex_editing.webm)
+For example see below: ![image: shape vertex editing](../../images/shape_vertex_editing.webm)
```
Comment (melissawm) [inline docs/howtos/layers/shapes.md]:
```suggestion
For example see below: ![animation of napari viewer with a shapes layer bounded by vertices over an initial static image. The shapes are selected and the vertices moved by the mouse pointer, resulting in new shapes.](../../images/shape_vertex_editing.webm)
```

**Review by melissawm — APPROVED** [review]
Thanks @Carreau !

**Comment by isabela-pf** [comment]
Just checking in, is there anything else I need to do to help get this merged? It looks good to me.

**Review by psobolewskiPhD — APPROVED** [review]

**Comment by psobolewskiPhD** [comment]
> Just checking in, is there anything else I need to do to help get this merged? It looks good to me.

Sorry! Slipped through the cracks I guess.
I've approved!

**Pull request merged**